CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Added `WithPeerServiceName` option to sarama instrumentation. ([#119](https://github.com/signalfx/signalfx-go-tracing/pull/119))
```

Comment:
```suggestion
- `WithPeerServiceName` option to sarama instrumentation. ([#119](https://github.com/signalfx/signalfx-go-tracing/pull/119))
```

Why?
1. `Added` is redundant
2. Usually the bullets are `Add` / `Fix` and not  `Added`/`Fixed` if really needed

Reply:
IDK, reading 

> `WithPeerServiceName` option to sarama instrumentation.

doesn't make sense to me. This is for humans to consume so I'd prefer to have sentences that make more sense than following changelog spec.